flink-core/src/main/java/org/apache/flink/configuration/CheckpointingOptions.java

```diff
@@ -142,8 +142,8 @@ public class CheckpointingOptions {
      * This option configures local recovery for this state backend. By default, local recovery is
      * deactivated.
      *
-     * <p>Local recovery currently only covers keyed state backends. Currently, MemoryStateBackend
-     * and HashMapStateBackend do not support local recovery and ignore this option.
+     * <p>Local recovery currently only covers keyed state backends. Currently, the
+     * OperatorStateBackend does not support local recovery and ignore this option.
```

Comment:
Could you also remove `OperatorStateBackend` from here please? (same as [here](https://github.com/apache/flink/pull/20708#discussion_r957727218)).

Reply:
Sure.